### PR TITLE
[Hotfix][FSI] Validation tests and warning fix

### PR DESCRIPTION
**Pull request by rubenzorrilla**
Fixes the error and the warning in  #3629 (thanks @philbucher for reporting).

### CODE REVIEW

File: applications/FSIapplication/python_scripts/fsi_analysis.py
Comment (philbucher) [inline applications/FSIapplication/python_scripts/fsi_analysis.py:81]: hm maybe we should make this protected such that you can access it

File: applications/FSIapplication/python_scripts/fsi_analysis.py
Comment (philbucher) [inline applications/FSIapplication/python_scripts/fsi_analysis.py:93]: If this stuff would be in a separate function then you could reuse more things, I will keep it in mind

File: applications/FSIapplication/python_scripts/fsi_analysis.py
Comment (philbucher) [inline applications/FSIapplication/python_scripts/fsi_analysis.py]: maybe its better to check against a small tolerance here

**Review by philbucher — COMMENTED** [review]
can you merge master and try if the test still works?
I merged the fix in the python solver #3375 today but you don't have this change in this branch

File: applications/FSIapplication/python_scripts/fsi_analysis.py
Comment (rubenzorrilla) [reply applications/FSIapplication/python_scripts/fsi_analysis.py:81]: This was my thought as well. I did this as a temporary solution to avoid modifying the Analysis Stage in this PR.

File: applications/FSIapplication/python_scripts/fsi_analysis.py
Comment (rubenzorrilla) [reply applications/FSIapplication/python_scripts/fsi_analysis.py]: That's true! I'll change it. Thanks!

**Comment by philbucher** [comment]
I will approve after you merged master and ran the tests again

**Comment by rubenzorrilla** [comment]
@philbucher Thanks. I'll do it ASAP.

**Comment by rubenzorrilla** [comment]
@philbucher I've merged the master and the Mok validation test runs as expected. However, the two sides mapping test fails... In principle this has nothing to do with the changes being discussed in here. I will have a look to it.

**Comment by rubenzorrilla** [comment]
@philbucher master merged and tests running (there also was a minor tolerance issue with one test that has nothing to do with the latest changes in Kratos). Can you approve?

**Review by philbucher — APPROVED** [review]
Nice!

One last comment: Can you check if there are still some compiler-warnings abt deprecated functions? If I am not mistaken the mapper was using some deprecated function from `condition`

**Pull request merged**
